versioning/scripts/dockerfiles/main.go

```diff
@@ -71,6 +72,17 @@ func deleteIfFileExists(path string) {
 	}
 }
 
+// this function removes all white characters from the string
```

Comment:
godoc convention: s/this/removeWhiteCharacters/

Reply:
OK, fixed.